queue_services/entity-emailer/pre_hook_create_database.py

```diff
@@ -0,0 +1,30 @@
+import contextlib
+import os
+import sys
+
+import sqlalchemy
+import sqlalchemy.exc
+
+from entity_emailer.config import ProdConfig
+
+DB_ADMIN_PASSWORD = os.getenv('DB_ADMIN_PASSWORD', None)
+DB_ADMIN_USERNAME = os.getenv('DB_ADMIN_USERNAME', 'postgres')
+
+if not hasattr(ProdConfig, 'TRACKER_DB_NAME') or not DB_ADMIN_PASSWORD:
+    print("Unable to create database.", sys.stdout)
+    sys.exit(-1)
+
+DATABASE_URI = 'postgresql://{name}:{password}@{host}:{port}/{name}'.format(
+    password=DB_ADMIN_PASSWORD,
+    host=ProdConfig.TRACKER_DB_HOST,
+    port=int(ProdConfig.TRACKER_DB_PORT),
+    name=DB_ADMIN_USERNAME,
+)
+
+with contextlib.suppress(sqlalchemy.exc.ProgrammingError):
```

Comment:
i added the autobuild to the scripts way early on.
we never deploy to DBs that don't already exist with the correct database available, so this can be left out

Reply:
deleted file